cmd/debos/debos.go

```diff
@@ -204,16 +204,6 @@ func main() {
 		return
 	}
 
-	if options.Verbose {
-		log.Println("Internal actions dump:")
-		DumpActions(reflect.ValueOf(r).Interface(), 0)
```

Comment:
I wonder if it's not the verify of the recipe action that should dump the actions after verify seems late.

Reply:
any thoughts on this?

Reply:
I think dump of the actions should be done after all recipes have been loaded, so after Verify has been called for all recipes.
This dumps the final list of all actions to do.
If this is done in Verify, it can be intermixed with the recipe print.

Reply:
On the other hand doing it this way  means the dump might not be printend at all of the Verify of an action fails, which is rather annoying as being able to use this to help debug is rather useful... I dont mind so much if the output would be the following:

<raw recipe of the root recipe>
<dump of the fileds of the root recip structs>
<raw recipe for first reecipe action>
<dump of fields first recipe action>
etc

the main key would be ensure the output is clear for which recipe the dumps are